cypress/fixtures/objects/moduleReqId/moduleReqId.json

```diff
@@ -2676,6 +2676,68 @@
                     }
                 ]
             }
+        },
+        "UserInterest": {
+          "Content.requestUserInterest - Positive Scenario: Platform-UX Notify requestUserInterest with type interest and reason playlist": {
+            "req": [
+              {
+                "method": {
+                  "id": "UserInterest 1.0",
+                  "description": "Positive Scenario: with type interest and reason playlist"
+                }
+              }
```

Comment:
since we have spec, the requirement id must match the spec document.

Reply:
we should be giving actual id like 3.1 , 3.2 etc

Reply:
corrected